File: C/StandardAlgo/Reculsive/Power/power.c
#include<stdio.h>
int power(int, int);
int main(void) {
	printf("%d\n", power(5, 2));
	return 0;
}
int power(int x, int n) {
	if (n == 0)return 1;
	else return x * power(x, n - 1);
}

File: Java/StandardAlgo/Algorithm/src/recursion/Power.java
package recursion;

public class Power {
	public static void main(String[] args) {
		System.out.println(power(2,2));
	}
	public static double power(int x, int n) {
		if(n == 0)
			return 1;
		else {
			return x*power(x,n-1);
		}
	}
}
